example/CMakeLists.txt

```diff
@@ -116,6 +116,18 @@ set_property(TEST PDI_example_decl_netcdf_C PROPERTY PROCESSORS 3)
 endif("${BUILD_NETCDF_PARALLEL}")
 endif("${BUILD_DECL_NETCDF_PLUGIN}")
 
+if("${BUILD_JSON_PLUGIN}")
+add_test(NAME PDI_example_json_C COMMAND "${RUNTEST_DIR}" "${MPIEXEC}" "${MPIEXEC_NUMPROC_FLAG}" 3 ${MPIEXEC_PREFLAGS} "$<TARGET_FILE:PDI_example_C>" ${MPIEXEC_POSTFLAGS} "${CMAKE_CURRENT_SOURCE_DIR}/json.yml")
```

Comment:
This example is lunch with 3 MPI process. But the json.yml in example ask for 4 process. For this reason, the test can be failed.

Suggestion: Added a test on mpi size as the tutorial to avoid this issue.